tests/ha/check_logs.pm

```diff
@@ -14,6 +14,7 @@ use testapi;
 use lockapi;
 use hacluster qw(get_cluster_name ha_export_logs);
 use version_utils 'is_sle';
+use utils;
```

Comment:
I don't think this is needed. `validate_script_output()` is in `testapi`, `validate_script_output_retry` is in `utils`.

Reply:
good catch, thanks.